### PR TITLE
Raise better errors during Google Auth faillure

### PR DESCRIPTION
**Pull request by NunoPinheiro**
Setting up two exceptions to help debugging when setting up Google authentication.
First exception happens when configuration is not set up properly. 
Second avoids a weird case where the user gets a valid logged in session, but is still considered anonymous.

### CODE REVIEW

File: security_monkey/sso/views.py [hunk 1]
```diff
@@ -291,6 +294,9 @@ def post(self):
         r = requests.get(people_api_url, headers=headers)
         profile = r.json()
 
+        if 'email' not in profile:
+            raise Exception("Unable to fetch user e-mail. Please ensure your application has Google+ API access")
```
Comment (mikegrima) [inline security_monkey/sso/views.py]: Would prefer a more specific exception class.

Comment (NunoPinheiro) [reply security_monkey/sso/views.py]: For both cases, or just for this exception?

Comment (mikegrima) [reply security_monkey/sso/views.py]: Both

Comment (mikegrima) [reply security_monkey/sso/views.py]: LGTM

**Pull request merged**